CONTRIBUTING.md

```diff
@@ -238,7 +238,7 @@ features for unmaintained components. At least one sponsor is needed to lift the
 adding the requested feature. Sponsors are expected to provide reviews for that feature and be responsive on the issue.
 
 Components marked as unmaintained still receive semantic conventions updates and bugfixes where possible.
-[@open-telemetry/javascript-triagers](https://github.com/orgs/open-telemetry/teams/javascript-triagers) may add the
+[@open-telemetry/javascript-contrib-triagers](https://github.com/orgs/open-telemetry/teams/javascript-contrib-triagers) may add the
```

Comment:
isn't here both cases js-contrib-triagers and js-triagers?

Reply:
In a separate Slack discussion I was proposing changing `js-contrib-triagers` *include* the `js-triagers` team -- in the same way that the current `js-triagers` team includes the `js-approvers` team, which includes the `js-maintainers` team.

If that is done, then just saying `js-contrib-triagers` should suffice.